lib/srv/reexec.go

```diff
@@ -165,10 +165,7 @@ func RunCommand() (io.Writer, int, error) {
 		errorWriter = tty
 		err = uacc.Open(c.UaccMetadata.UtmpPath, c.UaccMetadata.WtmpPath, c.Login, c.UaccMetadata.Hostname, c.UaccMetadata.RemoteAddr, tty)
 		if err != nil {
```

Comment:
Nit: Seems this "if" branch is basically empty now, save for the comment, so you could do instead:

```go
// uacc support is best-effort, ...
if err == nil {
  uaccEnabled = true
}
```

Reply:
true, fixed in latest commit